src/renderer/utils/forms/fields.ts

```diff
@@ -38,6 +52,6 @@ export const getNicknameField = (managedNodes?: Dict<Nickname>, ownNickname?: st
     .max(64, 'Nickname must not be more than 64 characters');
 };
 
-export const getPortField = () => yup.number().integer().required(REQUIRED_FIELD_ERROR);
+export const getPortField = () => {}
```

Comment:
do we still need this if we dont make it required?

Reply:
hmm. think we still need `yup.number()`

Reply:
i think it allows you to already to not set the port. But I don't know if it's just defaulting to 80

Reply:
think this is outdated - but its a bit tricky as using insecure http - it could be on any port